platforms/r3-corda-ent/configuration/roles/helm_component/templates/create_signer.tpl

```diff
@@ -0,0 +1,77 @@
+apiVersion: flux.weave.works/v1beta1
```

Comment:
this can be removed.. file has been renamed to signer.tpl

Reply:
also is this file required ?
configuration/roles/helm_component/templates/helm_component.tpl

Reply:
yes, thats the default template in case no match is found

Reply:
ok LGTM then